benchrun/python/README.md

```diff
@@ -0,0 +1,149 @@
+# Python {benchrun}
+
+{benchrun} is a Python package to run macrobenchmarks, deliberately designed to work
+well with the larger conbench ecosystem.
+
+## Installation
+
+{benchrun} is not [yet] on a package archive like PyPI; you can install from GitHub
+with
+
+```shell
+pip install benchrun@git+https://github.com/conbench/conbench.git@main#subdirectory=benchrun/python
```

Comment:
This did not work for me:
```
(base) elena@e benchmarks % pip install benchrun@git+https://github.com/conbench/conbench.git@edward/benchrun#subdirectory=benchrun/python
zsh: no matches found: benchrun@git+https://github.com/conbench/conbench.git@edward/benchrun#subdirectory=benchrun/python
```

Reply:
Hmm, it's worked for benchadapt in labs/benchmarks: https://github.com/voltrondata-labs/benchmarks/blob/main/requirements.txt#L4 Not sure why that would fail...

Reply:
wait exactly that does work for me: 

```shell
$ pip install benchrun@git+https://github.com/conbench/conbench.git@edward/benchrun#subdirectory=benchrun/python     
Collecting benchrun@ git+https://github.com/conbench/conbench.git@edward/benchrun#subdirectory=benchrun/python
  Cloning https://github.com/conbench/conbench.git (to revision edward/benchrun) to /private/var/folders/0j/zz6p_mjx2_b727p6xdpm5chc0000gn/T/pip-install-waj5bru7/benchrun_cd08123c40a9404c801db5eadf252c79
  Running command git clone --filter=blob:none --quiet https://github.com/conbench/conbench.git /private/var/folders/0j/zz6p_mjx2_b727p6xdpm5chc0000gn/T/pip-install-waj5bru7/benchrun_cd08123c40a9404c801db5eadf252c79
  Running command git checkout -b edward/benchrun --track origin/edward/benchrun
  Switched to a new branch 'edward/benchrun'
  branch 'edward/benchrun' set up to track 'origin/edward/benchrun'.
  Resolved https://github.com/conbench/conbench.git to commit fc5d2825aedf7f206a3affa1fbc55cfa5281a460
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: benchrun
  Building wheel for benchrun (setup.py) ... done
  Created wheel for benchrun: filename=benchrun-0.0.1-py3-none-any.whl size=15478 sha256=16740270c177f61198904471cffb9a9a0ca78a9acde276b7d726f24b40a5336c
  Stored in directory: /private/var/folders/0j/zz6p_mjx2_b727p6xdpm5chc0000gn/T/pip-ephem-wheel-cache-kuv39y27/wheels/51/99/10/8e53946c25765fba9c37082d375f22d4991383ee3abfa5391d
Successfully built benchrun
Installing collected packages: benchrun
  Attempting uninstall: benchrun
    Found existing installation: benchrun 0.0.1
    Uninstalling benchrun-0.0.1:
      Successfully uninstalled benchrun-0.0.1
Successfully installed benchrun-0.0.1
```
Maybe an ssh issue?

Reply:
...or possibly your pip is too old? What I used above: 

```bash
pip 22.2.2 from /Users/alistaire/mambaforge/envs/arrow-dev/lib/python3.9/site-packages/pip (python 3.9)
```

Reply:
I upgraded pip but same issue. You can ignore my comment. I found another way to install benchadapt and benchrun.